### ISSUE TITLE
Significance of temperature in UV packet?

### ISSUE THREAD

**Issue by martynwendon** [rfxcom/node-rfxcom#104]
More of a question than an issue as such, I added an Oregon UV sensor to my setup and note that it seems to be reporting temperature:

```
{
        "topic": "UV2/0x0600",
        "status": {
                "rssi": 5,
                "battery": 9
        },
        "payload": {
                "temperature": {
                        "value": 1510.4,
                        "unit": "degC"
                },
                "uv": {
                        "value": 0,
                        "unit": "UVIndex"
                }
        },
        "_msgid": "f060db34.837598"
}	
```    

Above output is from node-red-contrib-rfxcom but since all that's doing is passing through from node-rfxcom I thought the question belonged here.

So, where does the temperature come from?  Clearly it's not actually "temperature" else it would be a bit on the hot side :-)

Would be useful if it was a LUX reading perhaps?  It does seem to vary:

```
{
        "topic": "UV2/0x0600",
        "status": {
                "rssi": 5,
                "battery": 9
        },
        "payload": {
                "temperature": {
                        "value": 358.4,
                        "unit": "degC"
                },
                "uv": {
                        "value": 0,
                        "unit": "UVIndex"
                }
        },
        "_msgid": "5897e26.1fd3b1c"
}
```

Or is it just duff data?

**Comment by maxwellhadley** [comment]
This is a bug! Temperature should only be present in the packet for TFA (UV3) sensors. For other types, the temperature bytes in the message from the RFXtrx433 contain, as you have observed, rubbish.

It's been a long time, but it is now fixed. Expect a new version to be published some time "soon". I'm working through a backlog of stuff, most critically updating the very old version of serialport I have been using.

**Comment by maxwellhadley** [comment]
Fixed in npm version 2.6.0
